src/tsdl_image.ml

```diff
@@ -1,6 +1,11 @@
 open Ctypes
 open Foreign
 open Tsdl
+open Result
+
+type 'a result = 'a Sdl.result
```

Comment:
agreed -- happy to commit it in if it helps; but don't want to get in your way at this point